sparsity/test/test_sparse_frame.py

```diff
@@ -877,6 +883,69 @@ def test_drop_multiple_labels():
     np.testing.assert_array_equal(sf.data.todense(), correct)
 
 
+def test_label_based_indexing_col(sample_frame_labels):
+    key = ['A', 'B']
+    results = [
+        sample_frame_labels[key],
+        sample_frame_labels.loc[:, key],
+        sample_frame_labels.reindex(columns=key)
+    ]
+    for res in results:
+        np.testing.assert_array_equal(
+            res.data.todense(), np.identity(5)[:, :2])
+        assert (res.index == pd.Index(list('VWXYZ'))).all()
+        assert (res.columns == pd.Index(list('AB'))).all()
+
+
+def test_label_based_indexing_idx(sample_frame_labels):
+    key = ['X', 'Y', 'Z']
+    results = [
+        sample_frame_labels.loc[key],
+        sample_frame_labels.reindex(labels=key, axis=0),
+        sample_frame_labels.reindex(index=key)
+    ]
+    for res in results:
+        np.testing.assert_array_equal(
+            res.data.todense(), np.identity(5)[2:, :])
+        assert (res.index == pd.Index(['X', 'Y', 'Z'])).all()
+        assert (res.columns == pd.Index(list('ABCDE'))).all()
+
+
+def test_label_based_col_and_idx(sample_frame_labels):
+    key = ['V', 'W'], ['A', 'B']
+    results = [
+        sample_frame_labels.loc[key],
```

Comment:
I think indexing with `.loc[(A, B)]` is not necessarily the same as indexing with `.loc[A, B]`. Could you please check the second option? Actually, I'm surprised that the tuple works here (it works also in pd.DataFrame) because it's not mentioned here: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.loc.html . I thought it's reserved for multiindex.

Reply:
Hm actually it's both the same if you use the brackets internally the object's [__getitem__](https://docs.python.org/3.4/reference/datamodel.html#object.__getitem__) method will be called (which supports only a single argument) with the argument inside the brackets which in both cases evaluates to a tuple first.

Reply:
I'd rather stick to the `[A, B]` syntax just to be sure that we're independent of any internal stuff and follow the documented API.

Reply:
Well as said it's the same, but I agree:
```
>>> 1, 2 == (1, 2)
True
```
For the test it is ok though to leave it like this. Else I'd have to retype the key every time...

Reply:
Come on, I know that `1, 2 == (1, 2)`, but `foo(1, 2)` is not the same as `foo((1, 2))`.
And I'm not sure what you mean by "every time" here. Anyway, it's your decision what to do here.

Reply:
Hmmm yep but `.loc[]` is not really a function call :P

Yeah I didn't get your second comment then exactly

Reply:
Okay, nevermind.